src/modules/apps/utils.ts

```diff
@@ -144,3 +145,39 @@ export function optionsFormatter(billingOptions: BillingOptions) {
   table.push([{ content: chalk.bold('Terms of use:'), hAlign: 'center' }, { content: billingOptions.termsURL, hAlign: 'center' }])
   return table.toString()
 }
+
+export async function checkBuilderHubMessage(cliRoute: string): Promise<any> {
+  const http = axios.create({
+    baseURL: `https://calasans--basedevmkp.myvtexdev.com`,
+    timeout: 10000,
+  })
+  try {
+    const res = await http.get(`/_v/private/builder/0/getmessage/${cliRoute}`)
+    return res.data
+  } catch (e) {
+    return {}
+  }
+}
+
+const promptConfirm = (msg: string): Promise<string> =>
+  inquirer.prompt({
+    message: msg,
+    name: 'appName',
+    type: 'input',
+  })
+    .then<string>(prop('appName'))
+
+export async function showBuilderHubMessage(message: string, showPrompt: boolean, manifest: Manifest) {
+  if(message) {
+    if (showPrompt) {
+      const confirmMsg = `Are you absolutely sure?\n${message ? message : ''}\nPlease type in the name of the app to confirm (ex: vtex.getting-started):`
+      const appNameInput = await promptConfirm(confirmMsg)
+      const AppName = `${manifest.vendor}.${manifest.name}`
+      if (appNameInput !== AppName) {
+        throw new CommandError(`${appNameInput} doesn't match with the app name.`)
+      }
+    } else {
+      console.log(message)
```

Comment:
Rather than `console.log`, we should use the `log` (from `./src/logger`), right? (and then we may decide if we use `log.warn`, `log.info`, etc.